Solutions/MimecastTTP/Analytic Rules/MimecastTTPAttachment.yaml

```diff
@@ -0,0 +1,45 @@
+id: aa75944c-a663-4901-969e-7b55bfa49a73
+name: Mimecast Targeted Threat Protection - Attachment Protect
+description: Mimecast Targeted Threat Protection - Attachment Protect
+severity: High
+requiredDataConnectors:
+  - connectorId: MimecastTTPAPI
+    dataTypes:
+      - MimecastTTPAttachment_CL
+enabled: true
+query: MimecastTTPAttachment_CL| where result_s <> "safe";
+queryFrequency: 5m
+queryPeriod: 15m
+triggerOperator: gt
+triggerThreshold: 0
+suppressionDuration: 5h
+suppressionEnabled: false
+tactics:
+- InitialAccess
+- Discovery

```

Comment:
Please add relevantTechniques, Relevant [MITRE Techniques](https://attack.mitre.org/versions/v11/matrices/enterprise/) ID.

This needs to be added in all the analytical rules of the PR.

Example :- https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/Azure%20Active%20Directory/Analytic%20Rules/FailedLogonToAzurePortal.yaml